homeassistant/components/vallox/__init__.py

```diff
@@ -334,3 +346,35 @@ def __init__(self, name: str, coordinator: ValloxDataUpdateCoordinator) -> None:
             sw_version=self.coordinator.data.sw_version,
             configuration_url=f"http://{self.coordinator.config_entry.data[CONF_HOST]}",
         )
+
+
+@callback
+def _async_log_deprecated_service_call(
```

Comment:
We don't need a helper unless we need to create an issue for more than one service. We can inline the code.